internal/proto/local/controller/storage/credential/vault/store/v1/vault.proto

```diff
@@ -42,30 +42,30 @@ message CredentialStore {
   // complete URL.
   // It must be set.
   // @inject_tag: `gorm:"not_null"`
-  string vault_address = 8;
+  string vault_address = 8 [(custom_options.v1.mask_mapping) = {this:"VaultAddress" that: "attributes.address"}];
 
   // namespace is the vault namespace the credential store should use.
   // It is optional.
   // @inject_tag: `gorm:"default:null"`
-  string namespace = 9;
+  string namespace = 9 [(custom_options.v1.mask_mapping) = {this:"Namespace" that: "attributes.namespace"}];
 
   // ca_cert is a single PEM-encoded CA certificate to verify the Vault
   // server's SSL certificate.
   // It is optional.
   // @inject_tag: `gorm:"default:null"`
-  bytes ca_cert = 10;
+  bytes ca_cert = 10 [(custom_options.v1.mask_mapping) = {this:"CaCert" that: "attributes.vault_ca_cert"}];
 
   // tls_server_name is the name to use as the SNI host when connecting to
   // Vault via TLS.
   // It is optional.
   // @inject_tag: `gorm:"default:null"`
-  string tls_server_name = 11;
+  string tls_server_name = 11 [(custom_options.v1.mask_mapping) = {this:"TlsServerName" that: "attributes.tls_server_name"}];
 
   // tls_skip_verify disables verification of TLS certificates. Using this
   // option is highly discouraged as it decreases the security of data
   // transmissions to and from the Vault server.
   // @inject_tag: `gorm:"default:false"`
-  bool tls_skip_verify = 12;
+  bool tls_skip_verify = 12 [(custom_options.v1.mask_mapping) = {this:"TlsSkipVerify" that: "attributes.disable_tls_verification"}];
```

Comment:
`tls_skip_verify` was chosen to be consistent with the name [Vault](https://www.vaultproject.io/docs/agent#tls_skip_verify), [Consul](https://www.consul.io/docs/connect/ca/vault#tlsskipverify), [Nomad](https://www.nomadproject.io/docs/configuration/vault#tls_skip_verify), and [Terraform](https://registry.terraform.io/providers/hashicorp/vault/latest/docs#skip_tls_verify) use for this parameter. Why change it here?

Reply:
That is a good enough reason for me to keep it as `tls_skip_verify` :).  I wasn't aware of those existing fields, so I changed it to be similar to the flag we use to disable oidc config validation since there seemed to be comparable `disable_discovered_config_validation`. `